modules/aws_base/tf_module/existing_vpc.tf

```diff
@@ -0,0 +1,37 @@
+data "aws_vpc" "vpc" {
+  count = local.create_vpc ? 0 : 1
+  id    = var.vpc_id
+  state = "available"
+}
+
+data "aws_subnet" "public_subnets" {
+  for_each = local.create_vpc ? toset([]) : toset(var.public_subnet_ids)
+  id       = each.key
+  vpc_id   = var.vpc_id
+  # state    = "available"
```

Comment:
what was the story behind uncommenting the state for the various imported resources?
shall they be removed?

Reply:
^